### PR TITLE
Votes backup CTA on `VoteForCouncilSuccessModal` (#1633)

### LINKED ISSUE #1633: Expose and backup voting salt
## Background

Currently, the voting [salt](https://joystream.gitbook.io/joystream-handbook/governance/council#reveal-vote) is generated in the browser and stored in local storage. This is obviously not great, because it means that a browser state reset will cause you to loose your ability to reveal your vote, and you also cannot use a different browser or device.

## Proposal

We should make it possible for the user to backup and import their salt when voting. The backup could be to simply display the salt, or to allow a file download - as those likely have an easier time persisting. There also needs to be an input or import step during revealing.

## User Stories

**Backup salt**
- [ ] On the first vote member is displayed with a salt modal
   - Concept of salt (tooltip) "Voting salt is a random string, generated for a given member id and used for securing the vote to be published without exposing the contents of the vote, and contents to be securely revealed during reveal stage. It is stored locally in the browser and all votes submitted on this device and browser can be revealed using this salt. Voting salt is _required_ to reveal your vote. It can be lost forever, if cookies are cleared. Backup your voting salt to ensure your vote can be revealed later, even from a different browser or device."
   - Input for [meta data ](https://joystream.gitbook.io/joystream-handbook/governance/proposals#vote-1)(pre-populated; proposal; The type of vote; rationale; councilor)
   - Preview of filename (using meta inputs as a mask, editable)
   - "Display" Salt -> so can be copy-pasted ❓ 
    - "Checkbox" to acknowledge that vote can be lost if salt is lost, default value = false
   - "Checkbox" Export Voting Salt file, default set = true
   - CTA "view announcement"
   - 💡  would be great if there is some sort of self-validation in the export format, so that it describes what specific vote, voter member id, in what election and so on, the salt is for, so that when the user tries to import, they are warned if they have the wrong salt, and perhaps explains what it is for, e.g. this salt is not for your membership, this salt is not for this vote, etc.

Figma link to the [Vote Submitted Modal](https://www.figma.com/file/Z3KcRoAll3JGEY622o0iZu/Council?node-id=218%3A32146)

 - [ ] Subsequent votes from same device+browser could have a modification of this window, since salt is already stored by browser and may have been exported.

**Reveal Vote flow when Salt missing**
- [ ] User logged in from a diff browser and device shown an additional modal to import voting salt as part of [reveal vote flow](https://www.figma.com/file/Z3KcRoAll3JGEY622o0iZu/Council?node-id=218%3A27572) 
- [ ] User can add a string value of salt, instead of file import ❓ 
- [ ]  Validation errors e.g. this salt is not for your membership, this salt is not for this vote,
- [ ]  Salt is vote-specific, bulk salt import not supported.

**Recover**
- [ ] User can "recover" the salt if lost, but still logged in from the browser and device where vote was casted.
- [ ]  Salt backup (repeat backup of salt) can be done from the vote modal (as opposed to [top of the screen CTAs](https://www.figma.com/file/Z3KcRoAll3JGEY622o0iZu/Council?node-id=156%3A15411)) Shall we just add a tertiary link here? What's best based on design system?

### PR DESCRIPTION
**Pull request by traumschule**
Fixes #1633

### CODE REVIEW

File: packages/ui/src/council/modals/VoteForCouncil/VoteForCouncilSuccessModal.tsx [hunk 1]
```diff
@@ -30,9 +31,10 @@ export const VoteForCouncilSuccessModal = ({ onClose, candidateId }: Props) => {
 
       <SuccessModalBody>
         <TextMedium margin="l" light>
-          You have just successfully voted for the Candidate
+          You have just successfully voted for the Candidate. This is a good time to backup your votes.
         </TextMedium>
-
+        <BackupVotesButton cycleId={candidate?.cycleId} />
+        ￼
```
Comment (thesan) [inline packages/ui/src/council/modals/VoteForCouncil/VoteForCouncilSuccessModal.tsx:37]: @traumschule is this on purpose ?

**Review by thesan — CHANGES_REQUESTED** [review]
The tests fails because the `@testing-library` mocked version of `URL` is missing the `createObjectURL` method (I think it might be missing `revokeObjectURL`too). To fix the tests this method has to be mocked. I think it's done somewhere else in the codebase.

File: packages/ui/src/council/modals/VoteForCouncil/VoteForCouncilSuccessModal.tsx (hunk 1 above)
Comment (traumschule) [reply packages/ui/src/council/modals/VoteForCouncil/VoteForCouncilSuccessModal.tsx:37]: i added empty lines around `BackupVotesButton` for clarify but prettier had another view, or do you mean something else?

**Comment by traumschule** [comment]
```
yarn test election
yarn run v1.22.19
$ jest election
 PASS  test/council/components/ElectionTabs.test.tsx
 PASS  test/council/pages/PastElection.test.tsx (11.096 s)
 PASS  test/council/components/ElectionVotes.test.tsx (11.591 s)
 PASS  test/council/pages/Election.test.tsx (13.933 s)

Test Suites: 4 passed, 4 total
Tests:       43 passed, 43 total
Snapshots:   0 total
Time:        15.806 s
Ran all test suites matching /election/i.
Done in 16.71s.
```

**Review by thesan — APPROVED** [review]
I just moved the URL mocks in the jest `setup.ts`, so this shouldn't be a problem anymore

**Comment by traumschule** [comment]
forum and proposal tests failed, seems unrelated this time.

**Comment by dmtrjsg** [comment]
Pls update the copy to 

"Import your file with voting salt here and submit the transaction to restore votes."

Screenshot 2022-11-24 at 13 16 40

**Pull request merged**